src/lwc/geTemplateBuilderFormField/geTemplateBuilderFormField.html

```diff
@@ -9,6 +9,7 @@
                                          data-name="customLabel"
                                          field-level-help={labelHelpText}
                                          label={field.label}
+                                         data-id="formField"
```

Comment:
What do you think about using `data-name` instead of adding in `data-id`?

Reply:
@psadie-sfdo I used `data-id` here because `data-name` is already used in this input component.